src/strings/PowerFxResources.en-US.resx

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
>standalone [](http://example.com/codeflow?start=37&length=10)

Is this intentional change?

Reply:
no - just edited it via the default VS Resx editor.  I can revert this part.